spec.bs

```diff
@@ -973,7 +973,11 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       |config|["{{AuctionAdConfig/additionalBids}}"] until an associated [=request=] whose
       [=request/initiator type=] is `"fetch"` and the {{RequestInit/adAuctionHeaders}} option set to
       `true` returns a response, or alternatively, not calling {{Navigator/runAdAuction()}} until the
-      [=request=] returns a response.
+      [=request=] returns a response. Another alternative is to issue an
+      <a href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#navigate-an-iframe-or-frame">iframe navigation</a>
+      request with the <{iframe/adAuctionHeaders}> <a spec=html>content attribute</a> set to `true`,
+      calling {{Navigator/runAdAuction()}} after that request has returned a response; in this case,
+      the JavaScript code can immediately resolve |config|["{{AuctionAdConfig/additionalBids}}"].

```

Comment:
I think this works, but I'm wondering how we'll wait for the network load on the implementation side -- a design doc about how this works would be good.

Reply:
Note that this is a non-normative NOTE: -- I didn't see anything in the normative spec language that would guarantee this property, but I think we need that for this to actually work? Maybe that's actually what we need instead of a design doc?